pymc/distributions/continuous.py

```diff
@@ -1287,6 +1287,16 @@ def logcdf(value, a, b):
             msg="a > 0, b > 0",
         )
 
+    def icdf(value, a, b):
+        res = (1 - (1 - value) ** pt.reciprocal(b)) ** pt.reciprocal(a)
```

Comment:
Looking at this, would it help to compute this icdf on the log scale (to get rid of these exponents) and only exponentiate the final result?


```suggestion
        res = exp(log(1 - (1 - value) * pt.reciprocal(b) * pt.reciprocal(a))
```
Or something along those lines (might have made a mistake)

Reply:
Done.